airflow/config_templates/default_airflow.cfg

```diff
@@ -658,6 +658,18 @@ auto_refresh_interval = 3
 # Boolean for displaying warning for publicly viewable deployment
 warn_deployment_exposure = True
 
+# Number of dags to look back for dag audit log view
+audit_trail_query_days = 31
+
+# Types of events to ignore for dag audit log view
+# E.g. excluded_events = success,failed,skipped,landing_times
+excluded_events =
```

Comment:
Is it worth validating that `included_events` and `excluded_events` have no overlap? I know that its a weird edge case / user Error, but I think this would lead to invalid queries when rendering the view.

Reply:
hey @SamWheating it really is. Thank you for mentioning that. I will go ahead and add an update there!